protocols/smtp/src/main/java/org/apache/james/protocols/smtp/MailAddressException.java

```diff
@@ -19,6 +19,8 @@
 
 
 package org.apache.james.protocols.smtp;
+
+import java.util.List;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.util.List'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://172.17.0.2:9000/coding_rules#rule_key=squid%3AUselessImportCheck)